app/src/main/java/com/duckduckgo/app/survey/api/SurveyRepository.kt

```diff
@@ -74,6 +76,10 @@ class SurveyRepositoryImpl @Inject constructor(
         if (survey.daysInstalled == null && userBrowserProperties.daysSinceInstalled() < SURVEY_DEFAULT_MIN_DAYS_INSTALLED) {
             return false
         }
+        // Case for targeting all users
+        if (survey.daysInstalled == SURVEY_NO_MIN_DAYS_INSTALLED_REQUIRED) {
```

Comment:
Q: Not super familiar with Survey, how about the case where survey.daysInstalled is null? Or is that an impossible scenario when targetting all users?

Reply:
When we send null as `daysInstalled` it will be sent only to users with app install > 30 days. It is the previous check in line 76

Reply:
Yeah, like Noelia said, `survey.daysInstalled == null` in line 76 deals with `null` values.